src/StoreApi/Routes/Checkout.php

```diff
@@ -73,12 +73,6 @@ public function get_args() {
 					'context' => $this->get_context_param( [ 'default' => 'view' ] ),
 				],
 			],
-			[
-				'methods'             => WP_REST_Server::EDITABLE,
```

Comment:
I had to swap these around because the WP REST API for some reason would not do sanitisation and validation with editable first. I think it's because the methods overlap (POST).